atari-game-play/README.md

```diff
@@ -0,0 +1,46 @@
+# Atari - Solving Games with AI 🤖

```

Comment:
I'd encourage you to maybe follow some of the ideas in the `nba-pipeline` zenfile in terms of structure for the README.